app/tests/integration/nimbus/utils/filter_expression.js

```diff
@@ -0,0 +1,20 @@
+Components.utils.import("resource://gre/modules/components-utils/FilterExpressions.jsm");
+Components.utils.import("resource://gre/modules/components-utils/ClientEnvironment.jsm");
+
+// Targeting
+
+async function remoteSettings(arguments) {
+
+    const { TargetingContext } = ChromeUtils.import(
+        "resource://messaging-system/targeting/Targeting.jsm"
+    );
+    const ENVIRONMENT = {version: arguments[0], localeLanguageCode: arguments[1]};
```

Comment:
So here we're building an environment explicitly, but this won't be the same environment that Firefox desktop actually uses when a real experiment is launched.  

The goal of this test is to validate that the JEXL in each TargetingConfig will run as expected in Firefox, which includes both being syntactically valid (which is now checked in unit tests using PyJEXL) and that each clause in the expression will evaluate to a symbol in the environment and not undefined (ie if someone mispells a variable).

So we'll need to pull in the real environment that Firefox provides rather than specifying it here manually.

Reply:
Looks like this should do it:

https://searchfox.org/mozilla-central/source/toolkit/components/nimbus/lib/RemoteSettingsExperimentLoader.jsm#110-113

Reply:
So how do I do this?

Reply:
Well you probably just need to import the RemoteSettingsExperimentLoader module and call that method?

Reply:
yup, I added that comment before I refreshed the page.